app/scripts/lib/util.js

```diff
@@ -246,3 +247,48 @@ export function previousValueComparator(comparator, initialValue) {
     }
   };
 }
+
+export function formatTxMetaForRpcResult(txMeta) {
```

Comment:
This was migrated from the network controllers utils module to the general utils module, because it was no longer used in the network controller